src/devices/engo.ts

```diff
@@ -312,4 +312,109 @@ export const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint("TS0601", ["_TZE204_cmyc8g5i"]),
```

Comment:
This code looks very similar to the `EONE` definition that is also defined in this file, can you try re-using that one?